sim/midas/src/main/scala/midas/platform/VitisShim.scala

```diff
@@ -120,5 +121,6 @@ class VitisShim(implicit p: Parameters) extends PlatformShim {
     GoldenGateOutputFileAnnotation.annotateFromChisel(
       s"# Currenty unused",
       ".env.tcl")
+    SpecifyXDCCircuitPaths(Some("firesim_top"), Some("WRAPPER_INST/CL/firesim_top"))
```

Comment:
You probably know this already, but just as a note, these will need to be updated to reflect the instance path hierarchy in Vitis.

Reply:
Wasn't sure if it changed and needed to generate the Verilog to check. Thanks.